apps/web-giddh/src/assets/locale/all-features/hi.json

```diff
@@ -151,5 +151,24 @@
 	"support": "सहयोग",
 	"talk_to_sales": "सेल्स से बात करें",
 	"help_doc": "सहायता दस्तावेज़",
-	"schedule_demo": "डेमो शेड्यूल करें"
+	"schedule_demo": "डेमो शेड्यूल करें",
+    "purchase_bill": "Purchase Bill",
```

Comment:
Please add hindi translation here

Reply:
Not provided by sandhya, Will update once I get